repobuilder/job.go

```diff
@@ -223,6 +223,7 @@ func (j *repoBuilderJob) setup() {
 				j.NotaryKey = "richard"
 				j.NotaryToken = os.Getenv("NOTARY_TOKEN_DEB_LEGACY")
 			} else {
+				// TODO: maybe need to change this.
```

Comment:
Not sure what to do here, or if this even needs to be changed.

Reply:
There's no more stable release series (`NewMongoDBVersion` returns the empty string). You'll probably have to handle new and legacy versions differently here.

Reply:
Do we know what the notary key is used for? What should this be in the new version case?

Reply:
The notary key is necessary to sign the packages with the notary service.